src/notebooklm/cli/source.py

```
@@ -165,7 +165,7 @@ def source_list(ctx, notebook_id, json_output, client_auth):
     nb_id = require_notebook(notebook_id)
 
     async def _run():
-        async with NotebookLMClient(client_auth) as client:
+        async with NotebookLMClient(client_auth, timeout=timeout) as client:
```

Ruff (0.15.11)

[error] Undefined name `timeout`

(F821)